sigmastate/src/test/scala/sigmastate/JitCostingSpecification.scala

```diff
@@ -0,0 +1,444 @@
+package sigmastate
+
+import org.ergoplatform.SigmaConstants.ScriptCostLimit
+import org.ergoplatform.ErgoScriptPredef.TrueProp
+import org.ergoplatform.validation.ValidationRules
+import org.ergoplatform.{ErgoLikeContext, ErgoLikeTransaction, ErgoBox}
+import scorex.crypto.authds.avltree.batch.Lookup
+import scorex.crypto.authds.{ADDigest, ADKey}
+import scorex.crypto.hash.Blake2b256
+import sigmastate.Values.{TrueLeaf, BigIntConstant, AvlTreeConstant, ConstantPlaceholder, ByteArrayConstant, IntConstant, ErgoTree, BooleanConstant}
+import sigmastate.eval.Extensions._
+import sigmastate.eval.Sized._
+import sigmastate.eval._
+import sigmastate.helpers.{ContextEnrichingTestProvingInterpreter, ErgoLikeTestInterpreter}
+import sigmastate.interpreter.ContextExtension
+import sigmastate.interpreter.Interpreter.{ScriptNameProp, ScriptEnv, emptyEnv}
+import sigmastate.utxo.{CostTable, JitCostTable}
+import sigmastate.utxo.CostTable._
+import special.sigma.{SigmaTestingData, AvlTree}
+
+class JitCostingSpecification extends SigmaTestingData {
```

Comment:
To reviewer: 
This spec will be further updated (in separate PR as part of implementing #665)
The old CostingSpecification remains unchanged since the AOT version should remain unchanged.